src/classes/BotManager.ts

```diff
@@ -74,6 +74,10 @@ export = class BotManager {
         return this.bot !== null && this.bot.isReady();
     }
 
+    // reloadOptions(options: Options): void {
```

Comment:
reload options is just a loadOptions and then deepmerge into the bot.options if you want it

Reply:
nope since the current liveOptions things work fine. I am just about to remove this.